sdk/azcore/log_test.go

```diff
@@ -9,46 +9,48 @@ import (
 	"fmt"
 	"net/http"
 	"testing"
+
+	"github.com/Azure/azure-sdk-for-go/sdk/internal/logger"
 )
 
 func TestLoggingDefault(t *testing.T) {
 	// ensure logging with nil listener doesn't fail
-	Log().SetListener(nil)
-	Log().Write(LogRequest, "this should work just fine")
+	SetListener(nil)
```

Comment:
Wondering if these two test should be moved to the internal module?

Reply:
In azcore, you could add tests for SetListener and SetClassification.

Reply:
I like having these tests here because they do a good job of making sure the external functionality hasn't changed drastically.

Reply:
I see, ok makes sense.